paris/src/main/java/com/airbnb/paris/proxies/ViewProxy.kt

```diff
@@ -82,10 +84,25 @@ class ViewProxy(view: View) : BaseProxy<ViewProxy, View>(view) {
             if (margin != null) {
                 marginParams.setMargins(margin, margin, margin, margin)
             } else {
-                marginBottom?.let { marginParams.bottomMargin = it }
-                marginLeft?.let { marginParams.leftMargin = it }
-                marginRight?.let { marginParams.rightMargin = it }
-                marginTop?.let { marginParams.topMargin = it }
+                if (marginHorizontal?.let { it >= 0 } == true) {
```

Comment:
Why exclude negative values? It seems like it could lead to incorrect behavior.

Reply:
ah, I'm not sure about this. I got the idea from `ViewGroup` - the class ignores negative values for these attrs